package.json

```diff
@@ -59,23 +59,23 @@
     "exports-loader": "^0.6.3",
     "expose-loader": "^0.7.1",
     "file-loader": "^0.8.5",
-    "html-webpack-plugin": "^2.9.0",
+    "html-webpack-plugin": "^2.10.0",
     "http-server": "^0.9.0",
     "ie-shim": "^0.1.0",
     "imports-loader": "^0.6.5",
     "istanbul-instrumenter-loader": "^0.2.0",
     "json-loader": "^0.5.4",
-    "karma": "^0.13.21",
+    "karma": "^0.13.22",
     "karma-chrome-launcher": "^0.2.2",
-    "karma-coverage": "^0.5.3",
+    "karma-coverage": "^0.5.4",
     "karma-jasmine": "^0.3.7",
-    "karma-mocha-reporter": "^1.1.6",
+    "karma-mocha-reporter": "^2.0.0",
     "karma-phantomjs-launcher": "^1.0.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "1.7.0",
-    "parse5": "^1.3.2",
-    "phantomjs-polyfill": "0.0.1",
-    "phantomjs-prebuilt": "^2.1.4",
+    "parse5": "^2.1.5",
```

Comment:
we can't update this because angular2 requires `1.x.x` since they haven't updated it